Makefile

```diff
@@ -65,7 +65,46 @@ python_sdk::
 		rm ./bin/setup.py.bak && \
 		cd ./bin && python3 setup.py build sdist
 
+gen_examples::
+	cd ${WORKING_DIR}/examples/yaml && \
+	rm -rf ${WORKING_DIR}/examples/{go,nodejs,python,dotnet} && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language go --out ${WORKING_DIR}/examples/go 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language nodejs --out ${WORKING_DIR}/examples/nodejs 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language python --out ${WORKING_DIR}/examples/python 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language dotnet --out ${WORKING_DIR}/examples/dotnet 2>&1
+
+pulumi_login::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	pulumi login --local
+
+pulumi_up::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	cd ${WORKING_DIR}/examples/yaml && \
+	pulumi stack init dev && \
+	pulumi stack select dev && \
+	pulumi config set name dev && \
+	pulumi up -y
+
+pulumi_down::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	cd ${WORKING_DIR}/examples/yaml && \
+	pulumi stack select dev && \
+	pulumi destroy -y && \
+	pulumi stack rm dev -y
+
+devcontainer::
+	git submodule update --init --recursive .devcontainer && \
+	git submodule update --remote --merge .devcontainer && \
+	cp -f .devcontainer/devcontainer.json .devcontainer.json
+
 .PHONY: build
+
+up:: pulumi_login pulumi_up
+
+down:: pulumi_login pulumi_down
+
+deploy:: pulumi_login pulumi_up pulumi_down
```

Comment:
I don't see how setting up and then tearing down a stack is a `deploy`ment. I would re-think this name. That said, I'm not sure what this target should actually be used for, so maybe it is.

Reply:
i can remove the `deploy` target, I'm not extremely attached to it.

Reply:
These are now removed in https://github.com/pulumi/pulumi-provider-boilerplate/pull/80/commits/1ea1736d061ac57fa6ea7ee69311977061f45c10